pkg/controller/user/index.go

```diff
@@ -17,33 +17,107 @@ package user
 import (
 	"context"
 	"net/http"
+	"strconv"
 
 	"github.com/google/exposure-notifications-verification-server/pkg/controller"
 	"github.com/google/exposure-notifications-verification-server/pkg/database"
 )
 
+// pageSize is const for now, could provide options in UX
+const pageSize = 25
+
+type PageLabel struct {
+	Offset int
+	Label  string
+}
+
+type Pages struct {
+	Previous int
+	Current  int
+	Next     int
+	Offsets  []PageLabel
+}
+
 func (c *Controller) HandleIndex() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		ctx := r.Context()
+		qvar := r.FormValue("offset")
+		offset, _ := strconv.Atoi(qvar)
 
 		realm := controller.RealmFromContext(ctx)
 		if realm == nil {
 			controller.MissingRealm(w, r, c.h)
 			return
 		}
 
-		users, err := realm.ListUsers(c.db)
+		count, err := realm.CountUsers(c.db)
```

Comment:
Hmm, this is pretty expensive (count is a full table lock). I feel like we should have a `perPage` constant and then just pipe the current count into a SQL OFFSET